descriptors/ModuleDescriptor-template.json

```diff
@@ -378,6 +392,30 @@
             "circulation-storage.circulation-rules.get"
           ]
         },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/circulation/rules/request-policy",
```

Comment:
I think the addition of a new API should cause the version number to be bumped.

Reply:
Incremented it to version number to 6.1

Reply:
This needs to be circulatin 7.2, as master is now 7.1

Reply:
Oops :)

Reply:
Updated to 7.2